### DISCUSSION TITLE
Svelte/kit fastify adapter

### DISCUSSION THREAD

**Discussion by MrWaip** [sveltejs/kit#10981] [Ideas]
Hi. You may be interested in

Inspired by @sveltejs/adapter-node, I wrote an adapter for fastify

https://www.npmjs.com/package/@mrwaip/adapter-fastify?activeTab=readme
https://github.com/MrWaip/adapter-fastify/tree/main/packages/adapter

**Comment by nioc** [comment]
Hello, thanks for this adapter, does it will be officially referenced on [sveltesociety.dev](https://sveltesociety.dev/components#adapters)?
It seems to me that fastify has become a must for exposing APIs, it would be great to be able to do this directly from a sveltekit app.

**Reply by MrBns** [reply]
But there is nothing about Fastify..
